doc/specs/vulkan/chapters/initialization.txt

```diff
@@ -54,12 +54,12 @@ be cast to the type of the command being queried.
 
 1::
 The returned function pointer must: only be called with a dispatchable object
-(the first parameter) that is a child of pname:instance.
+(the first parameter) that is pname:instance or a child of pname:instance.
     e.g. sname:VkInstance, sname:VkPhysicalDevice, sname:VkDevice, sname:VkQueue, or
     sname:VkCommandBuffer.
 
 2::
-available extension is an extension function supported by any of the loader, ICD or layer.
+An \`\`available extension'' is an extension function supported by any of the loader, ICD or layer.
```

Comment:
Not supposed to be escaped:
https://github.com/KhronosGroup/Vulkan-Docs/blob/1.0/doc/specs/vulkan/style/markup.txt#L70